sql/core/src/test/scala/org/apache/spark/status/api/v1/sql/SqlResourceSuite.scala

```diff
@@ -41,31 +42,32 @@ object SqlResourceSuite {
 
   val nodeIdAndWSCGIdMap: Map[Long, Option[Long]] = Map(1L -> Some(1L))
 
+  val defaultAggregateMethod = SQLMetrics.defaultAggregateMethod("")
   val filterNode = new SparkPlanGraphNode(1, FILTER, "",
-    metrics = Seq(SQLPlanMetric(NUMBER_OF_OUTPUT_ROWS, 1, "")))
+    metrics = Seq(SQLPlanMetric(NUMBER_OF_OUTPUT_ROWS, 1, "", defaultAggregateMethod)))
```

Comment:
Changes in this class won't be needed if you add the default for the aggregateMethod.

Reply:
The default aggregate method is defined in `SQLMetrics`. For code structure, it is far from the UI class `SQLPlanMetric` and also conceptionally I don't think we should have a default aggregate method at `SQLPlanMetric`.